package.json

```diff
@@ -7,16 +7,16 @@
   "author": "Michał Pierzchała <thymikee@gmail.com>",
   "license": "BSD-3",
   "dependencies": {
-    "jest-zone-patch": "^0.0.7",
-    "jsdom": "^11.3.0",
-    "ts-jest": "^21.0.0"
+    "jest-zone-patch": "^0.0.8",
+    "jsdom": "11.5.1",
+    "ts-jest": "21.2.4"
   },
   "devDependencies": {
-    "jest": "^21.0.1",
-    "typescript": "^2.3.0"
+    "jest": "21.2.1",
+    "typescript": "~2.4.2"
```

Comment:
Why `~`? Is there something wrong with e.g `2.6`?

Reply:
At the moment Angular 5.0.x only supports up to 2.4.2, in the release 5.1.x they announced to support 2.5. Therefore I want to keep it consistent with Angular version supports. 5.0.x is stable now in my opinion so I'd like to keep TypeScript to be at the version supported by Angular 5.0.x. We can update to 2.5 later when Angular 5.1.x is stable.

Reply:
Cool, thanks for clearing this up!